### ISSUE TITLE
No record when Editor changes recommendation by Reviewer

### ISSUE THREAD

**Issue by pmangahis** [pkp/pkp-lib#8005]
**Describe the problem you would like to solve**
In the event, that an Editor changes the Reviewer's Recommendation (in Reviewer details). There does not appear to be a record of the change in the UI. The decision gets automatically changes on both the Editor and Reviewers' dashboard.

In OJS 2, the email notification the reviewer sends does show the Reviewer's original decision in the Email log. 3 OJS 3.3 no longer sends this type of notification and does not get recorded in the Activity log either.

Steps to reproduce the behaviour:
1. Go to a completed review by clicking the blue arrow beside the reviewer's name followed by Review details. 
2. Adjust the Reviewer Recommendation
3. Decision that appears in the same pop-up box gets updated with the new recommendation by the Reviewer.
4. Look in the Activity log, no record made there
5. Log in as Reviewer, recommendation set by the Editor automatically gets applied.

**Describe the solution you'd like**
 To disable the edit of the reviewer recommendation (small change) and if editors are allowed to add reviews, then this should be shown in the log very clearly for transparency.

**Who is asking for this feature?**
Journal Editors

**Additional information**
Concerns from OJS 3.3 user that if the editor does do this (and does not have a record of the reviewer requesting the change) and having no record of the change being done it would be bordering on unethical/fraud.

There are some use cases identified on slack where an editor would need to maintain this functionality. Some editors have asked that they could add the whole review on behalf of the reviewer. These are cases where the reviewer is just not able to use the system and sends stuff with email. The editors still want the data to be stored in OJS.

**Comment by NateWr** [comment]
@pmangahis this is more of a feature request. Can you rephrase the issue description to match the feature request template? My initial reaction is that we definitely want to log this in the activity log and *maybe* want to notify the reviewer by email.

```
**Describe the problem you would like to solve**
Example: Our editors need a way to [...]

**Describe the solution you'd like**
Tell us how you would like this solution to be solved.

**Who is asking for this feature?**
Tell us what kind of users are requesting this feature. Example: Journal Editors, Journal Administrators, Technical Support, Authors, Reviewers, etc.

**Additional information**
Add any other information or screenshots about the feature request here.
```

**Comment by NateWr** [comment]
After looking at the code, a log entry should be added any time the editor changes the review recommendation. This is `SUBMISSION_LOG_REVIEW_RECOMMENDATION_BY_PROXY`. We'll need to investigate to confirm if this is not happening and fix it.
